### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by InspectorGadget**

### CODE REVIEW

File: Plugin/New-API/AlwaysSpawn/src/AlwaysSpawn/Main.php [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Main extends PluginBase implements Listener{
 
     public function onEnable(){
         $this->getServer()->getPluginManager()->registerEvents($this, $this);
-        $this->getLogger()->log("[INFO] AlwaysSpawn Loaded!");
+        $this->getLogger()->info("[INFO] AlwaysSpawn Loaded!");
```
Comment (SOF3) [inline Plugin/New-API/AlwaysSpawn/src/AlwaysSpawn/Main.php]: Duplicate `[INFO]`

File: Plugin/New-API/AlwaysSpawn/src/AlwaysSpawn/Main.php [hunk 2]
```diff
@@ -22,11 +22,12 @@ public function onEnable(){
      * @priority       NORMAL
      * @ignoreCanceled false
      */
+    
     public function onSpawn(PlayerJoinEvent $event){
-        $player->teleport($this->level->getSpawn());
+        $event->getPlayer()->teleport($event->getPlayer()->getLevel()->getSafeSpawn());
```
Comment (SOF3) [inline Plugin/New-API/AlwaysSpawn/src/AlwaysSpawn/Main.php:27]: getSafeSpawn() or getSpawnLocation()?

Comment (InspectorGadget) [reply Plugin/New-API/AlwaysSpawn/src/AlwaysSpawn/Main.php:27]: I'm not familiar with getSpawnLocation(), is it set using /setdefaultspawn?

File: Plugin/New-API/AlwaysSpawn/src/AlwaysSpawn/Main.php (hunk 1 above)
Comment (InspectorGadget) [reply Plugin/New-API/AlwaysSpawn/src/AlwaysSpawn/Main.php]: ok, will address..

**Pull request closed without merging**